### ISSUE TITLE
Add meatal stenosis

### ISSUE THREAD

**Issue by jxchong** [obophenotype/human-phenotype-ontology#4246]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label

meatal stenosis

### 2. Synonyms

narrowing of the meatus

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

Meatal stenosis is the narrowing of the meatus, which is the opening on the tip of the penis. 

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

HP:0000036

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

unknown

**Comment by pnrobinson** [comment]
This is term HP_0032077, Male urethral meatus stenosis
The primary label is because there are other meatus's in the body, e.g., external meatus of the ear, and the term label needs to be unambiguous. Meatus stenosis almost always affects males, and so we put the term in the hierarchy for male external genital abnormalities. Meatal stenosis is an exact synonym.
